proto/buf/alpha/registry/v1alpha1/webhook.proto

```diff
@@ -104,6 +104,8 @@ message Webhook {
   string repository_name = 5;
   // The webhook repository owner name.
   string owner_name = 6;
-  // The subscriber's callback URL where notifications are delivered.
+  // The subscriber's callback url prefix (e.g. `https://example.com/route/here`).
+  // The input will be prefixed by the webhook event RPC
+  // (e.g. the result will be `https://example/route/here/buf.alpha.webhook.v1alpha1.EventService/Event`).
```

Comment:
Nit: I feel like its's worth calling out that we will use unary connect+json (maybe a link to the connect docs?) to call the `buf.alpha.webhook.v1alpha1.EventService/Event` rpc at this URL. The example is helpful, but the example shouldn't be the only thing that is telling me what the protocol/encoding/message type will be.

Reply:
We won't use JSON but connect+proto from [Peter's feedback](https://github.com/bufbuild/buf/pull/1249#discussion_r913959823).